content/behind_the_scenes/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Behind The Scenes"
+menu:
+  main:
+    weight: 1
+showDate: false
+draft: false
+---
+
+## Overview ##
+
+This article aims to explain how we have re-designed and re-written the backend used to collect, enrich and display the ADSB data our community sends us. 
+
+## Anatomy of a Feeder Connection ##
+When you follow our [docker_planewatch] 
```

Comment:
When you run/deploy our [Planewatch docker container/software](https://github.com/plane-watch/docker-plane-watch)

Reply:
I'll update this to point to the getting started guide on this site which also links to that GH page.